lib/roo/open_office.rb

```diff
@@ -1,5 +1,3 @@
-# frozen_string_literal: true
-
```

Comment:
For reduction of memory utilization, I think it should be placed.
How about replace `gsub!` with `gsub` ?